Package.swift

```diff
@@ -26,7 +26,7 @@ let package = Package(
         .library(name: "Navigation", targets: ["Navigation"]),
     ],
     dependencies: [
-        .package(url: "https://github.com/duckduckgo/duckduckgo-autofill.git", exact: "7.0.1"),
+        .package(url: "https://github.com/duckduckgo/duckduckgo-autofill.git", revision: "2fa8222"),
```

Comment:
Pointing to in-review branch -- https://github.com/duckduckgo/duckduckgo-autofill/pull/317
This will be updated before merging to an autofill release